joomla/joomla-cms
# Calendar css is broken #14195

# Test instructions

Use a 3rt party component that is using jhtml.calendar like kunena. (backend - kunena - statistics - click calendar)

You should see now the numbers and text on the popup.
